docs/views/home.handlebars

```diff
@@ -127,6 +127,10 @@
   <m-alert type="<%= alert.type %>"><%= alert.message %></m-alert>
 
 
+  <!--   htmx   -->
+  <m-alert hx-delete="/alerts" hx-swap="outerHTML">My Alert [dismiss]</m-alert>
```

Comment:
Hey, sorry for the late response. For this example I'm trying to show the same thing across all frameworks, i.e. an `alert` object in the framework's context (whatever that might be) and how it's used to set the Alert's type attribute and inner html.


Something like `<m-alert hx-delete="/alerts" hx-swap="outerHTML" type="{alert.type}">{alert.message}</m-alert>` to show that would be good

Reply:
@1cg just need that part done and I will happily merge

Reply:
@1cg please see above. I'll close in a few days if I don't hear anything. Feel free to reopen.